dds/DCPS/DataSampleHeader.cpp

```diff
@@ -458,6 +457,149 @@ DataSampleHeader::join(const DataSampleHeader& first,
   return true;
 }
 
+OPENDDS_STRING to_string(const MessageId value)
```

Comment:
These are all string literals, should we just use const char* here instead of an allocated string?